libs/libmilter/Makefile

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright (C) 2017 Val Kulkov <val.kulkov@gmail.com>
+#
+# This is free software, licensed under the MIT License
+#
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sendmail
```

Comment:
Yes and no.  It's part of the sendmail distribution, but it compiles into a standalone library that other MTA's like Postfix can also use.  I would have a sendmail package, but have subpackages like sendmail-libmilter.